site/specs-temp/numbers.json

```diff
@@ -17816,7 +17819,260 @@
         "required": [
           "eventType"
         ]
+      },
+      "AccountResponse": {
+        "type": "object",
+        "properties": {
+          "Account": {
+            "$ref": "#/components/schemas/Account"
+          },
+          "AddressErrorDescription": {
+            "$ref": "#/components/schemas/AddressErrorDescription"
+          }
+        }
+      },
+      "Address" : {
```

Comment:
We might want descriptions and examples here as well as nullable and default values(if any) defined

Reply:
Examplex, nullable default values for response schema?